src/assets/css/styles.css

```diff
@@ -159,4 +169,60 @@ menu > li:has(> menu)::after {
 }
 menu > li > a > img {
     vertical-align: middle;
-}
\ No newline at end of file
+}
+
+/* Social media icon colors */
```

Comment:
Is this just some AI code, or did you specifically want to change this stuff?

Reply:
I specifically did change the icons to all be from the same package so that they had a more consistent look (they are from https://simpleicons.org/). If we want the original ones do you know where they came from so we can try to consistently style them?

Reply:
I don't recall, probably just off of Google.

I was mainly wondering because of all of the !important tags which are usually a red flag for me. I think this is fine for now

Reply:
I'll do a pass at the `!important` ones again. Some of them were required so they are ignored when the dark theme gets used and it looks correct on the light version.